lib/foreman_virt_who_configure/engine.rb

```diff
@@ -73,10 +73,11 @@ class Engine < ::Rails::Engine
 
     # Include concerns in this config.to_prepare block
     config.to_prepare do
-      begin
-      rescue => e
-        Rails.logger.warn "ForemanVirtWhoConfigure: skipping engine hook (#{e})"
-      end
+      # begin
+        SSO::METHODS.unshift SSO::BasicWithHidden
+      # rescue => e
+      #   Rails.logger.warn "ForemanVirtWhoConfigure: skipping engine hook (#{e})"
+      # end
```

Comment:
I'm not sure if we won't need it in future but I'm ok with complete removal. I don't like masking problems.